lib/services/automationManagement/lib/automationClient.d.ts

```diff
@@ -22,6 +45,10 @@ declare class AutomationClient extends AzureServiceClient {
    *
    * @param {string} subscriptionId - Gets subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.
    *
+   * @param {string} automationAccountName - The name of the automation account.
+   *
+   * @param {string} automationAccountName1 - The name of the automation account.
```

Comment:
This isn't right. I don't see a change in the spec that would cause this redundant parameter to be created, so there might be something wrong in the generator or modeler.